src/templates/DataTable/pagination.eta

```diff
@@ -0,0 +1,110 @@
+import { Table } from "@tanstack/react-table";
+import {
+  Pagination,
+  PaginationContent,
+  PaginationItem,
+  PaginationLink,
+  PaginationNext,
+  PaginationPrevious,
+} from "@/components/ui/pagination";
```

Comment:
Please format the imports also so it works with a different alias.
I got that part working if u want some code. I'll push it later onto my fork in a new branch.

Reply:
good catch, thanks!